### DISCUSSION TITLE
How long will github reply a ticket? I was in pain waiting for a reply to my ticket.

### DISCUSSION THREAD

**Discussion by TCIano** [community/community#48340] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My friend @zonemeen, his account was suspended 25 days ago and his ticket #1989881 was created 24 days ago, no response, nothing. 
[A bro who had his account suspended on the same day has received a reply and reinstated his account](https://github.com/orgs/community/discussions/47041#discussioncomment-5075129). Do the official people know what is going on? My friend never violate TOS. This account is too important to him, he was in pain waiting for a reply to his ticket. Thanks

**Comment by TCIano** [comment]
Hi Lili, @queenofcorgis, Can you help me confirm that the ticket is in the right place? Is it in the normal processing flow? Thanks

**Reply by queenofcorgis** [reply]
Hi there - can confirm their ticket is in the right place - tickets are answered in the order they were received based on the team's resources. It looks like there are quite a few tickets ahead of theirs so if they could be please be patient ❤️ . Appreciate it!

**Reply by TCIano** [reply]
Hi Lili, @queenofcorgis, Thank you very much for your reply ❤️. Can you tell me approximately how many tickets are left in front of me and how long I need to wait?

**Reply by queenofcorgis** [reply]
Hi there - as it depends on the complexity of the tickets ahead of you and our team's resources - I can't provide that estimate. I'm only able to confirm that it is in the right place.

**Reply by TCIano** [reply]
Hi Lili, @queenofcorgis, Can you flag my ticket #1989881 in case it gets missed? I've been waiting too long 😢. Thank you very much ❤️.

**Reply by queenofcorgis** [reply]
HI there - I'm unable to interfere in our ticketing process as that is a separate team, I know this has been a long wait so I'll pass the feedback to the team.

**Reply by TCIano** [reply]
Thank you Lili @queenofcorgis. Does the team have any replies to your feedback?❤️
